src/component/veilederverktoy/avsluttoppfolging/avslutt-oppfolging.tsx

```diff
@@ -44,9 +44,11 @@ function AvsluttOppfolging() {
     }
 
     if (!avslutningStatus?.kanAvslutte) {
-        if (!avsluttOppfolgingOpptelt) {
-            logMetrikk(`veilarbvisittkortfs.metrikker.Avslutt_oppfolging_ikke_mulig`);
-            setAvsluttOppfolgingOpptelt(true);
+        if (avslutningStatus?.kanAvslutte !== undefined && !avslutningStatus?.kanAvslutte !== null) {
```

Comment:
Er det et utropstegn for mye her? 🤔 
```suggestion
        if (avslutningStatus?.kanAvslutte !== undefined && avslutningStatus?.kanAvslutte !== null) {
```

Reply:
Har rettet det.